students/swietczak_monika/lesson_12_debugging/regex_validators.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+
+def email_validator(some_email):
+    email_regex = re.compile(r"[^@]+@[^@]+\.[^@]+")
```

Comment:
It's more complicated than that. Not all the symbols are allowed. In real life you should search Google for "regex library" and get a comprehensive checker or use some builtins. In less real life situations you should take care that you accept a reasonable subset with `_` `.` etc., but not too much. 

The input you accepted could cause some trouble when. For example didn't put `$` at the end, so the potential attacker could put anything in the content of the mail message you are sending. You also accept spaces/inequality characters, semicolons, all rather dangerous in this context. `[\w\d_.-]+` `[a-z0-9_.-]+` would be much better than `[^@]+` here. In some places the `[^` construct is the best way to go and good that you know it.